crates/graphql/src/schema/strings.rs

```diff
@@ -0,0 +1,55 @@
+macro_rules! strings {
```

Comment:
Talked in huddle but my preference is to fork the data loader and add result support vs building a macro. Like people to read vanilla docs and be able to contribute without having to lookup custom macros.

Reply:
This one's actually not related to data loaders.  The macro there is just a bunch of repetitive struct and trait declarations for a bunch of near-identical structs, so it's only called once to declare all of them.  The only time it's called is [here](https://github.com/holaplex/solana-indexer/pull/171/files/59a8c32b05f2adcb54600fd6b557bd540cb6cde8#diff-2e1bcce04320328d2428e0eb8c7b392180b7264a80ce99a157666f9ecbfe9b82R50-R55) to declare a bunch of structs in one go.